src/AirshipActions.ts

```diff
@@ -9,15 +10,13 @@ export class AirshipActions {
   /**
    * Runs an Airship action.
    *
-   * @param name The name of the action.
-   * @param value The action's value.
+   * @param action The Airship Action.
    * @return A promise that returns the action result if the action
    * successfully runs, or the Error if the action was unable to be run.
    */
   public run(
```

Comment:
You can avoid this breaking change by keeping the same interface that we have now and wrap the values to the module call:

```
return this.module.actionRun({"name": actionName, "value": actionValue});
```